ml-conversational-analytic-tool/test_commentAnalysis.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+from commentAnalysis import CommentAnalyzer
+
+
+class TestCommentAnalysis(unittest.TestCase):
+
+    
+    def test_CommentAnalyzer_init(self):
+        analyzer = CommentAnalyzer(['test'])
+        """ positive test """
+        self.assertEqual(analyzer.word_count, {'test': 0})
+        """ negative test """
+        self.assertNotEqual(analyzer.word_count, {'test': 1})
+
+    def test_analyze_comments(self):
```

Comment:
The naming scheme for tests is "test_" + <name of the function>.

Reply:
Done